src/DotNetWorker.Grpc/Handlers/IInvocationHandler.cs

```diff
@@ -15,9 +16,9 @@ internal interface IInvocationHandler
         /// an associated cancellation token source for the invocation.
         /// </summary>
         /// <param name="request">Function invocation request</param>
-        /// <param name="enableUserException">flag to enable unwrapping user exception</param>
+        /// <param name="workerOptions"></param>
         /// <returns><see cref="InvocationResponse"/></returns>
-        Task<InvocationResponse> InvokeAsync(InvocationRequest request, bool enableUserException = false);
+        Task<InvocationResponse> InvokeAsync(InvocationRequest request, IOptions<WorkerOptions>? workerOptions = null);
```

Comment:
Not sure if this is a back and forth from a comment someone else left, but consider dropping the `IOptions` wrapper here and on the other similar methods.

```suggestion
        Task<InvocationResponse> InvokeAsync(InvocationRequest request, WorkerOptions? workerOptions = null);
```

The benefit is that there are 3 different forms of options - `IOptions`, `IOptionsSnapshot`, and `IOptionsMonitor` - we wouldn't want to force one of the 3.